libs/utils/utils.js

```diff
@@ -1157,3 +1157,9 @@ export function loadLana(options = {}) {
 }
 
 export const reloadPage = () => window.location.reload();
+
+export async function fetchMaybeCache(path) {
```

Comment:
Definitely a personal nit, but I can't say I'm crazy about the name 😅 How about `fetchWithCacheRules` or something of the likes to make the purpose clear?

Reply:
Totally fair haha. I was drawing a blank when I wrote it, I like your version better.